modified latte package.py to install interface to sedacs. also fixed bug: latte was not building with mpi.

finkeljos opened the pull request:
…bug, was not building with mpi properly.

<!--  
Remember that `spackbot` can help with your PR in multiple ways:
- `@spackbot help` shows all the commands that are currently available
- `@spackbot fix style` tries to push a commit to fix style issues in this PR
- `@spackbot re-run pipeline` runs the pipelines again, if you have write access to the repository 
-->

finkeljos committed:
modified latte package.py to install interface to sedacs. also fixed bug, was not building with mpi properly.

finkeljos committed:
make style edits. simplify blas/lapack vars for cmake

jeanlucf22 commented:
> @jeanlucf22 Can you take a look at these changes? I added latte branch we are working with that has interfacing capability to QMD software we are building. Also fixed bug with mpi build, you had -DO_MPI instead of -DDO_MPI for the mpi cmake flag.

Looks fine to me. Did I write the original script?

finkeljos commented:
> > @jeanlucf22 Can you take a look at these changes? I added latte branch we are working with that has interfacing capability to QMD software we are building. Also fixed bug with mpi build, you had -DO_MPI instead of -DDO_MPI for the mpi cmake flag.
> 
> Looks fine to me. Did I write the original script?

Yes, I think so.

finkeljos commented:
@haampie Can you complete the PR?

haampie commented:
Hi @finkeljos, this package-related pull request can be migrated from `spack/spack` to `spack/spack-packages` using the [**migrate-pkg-prs utility**](https://github.com/spack/migrate-package-prs).

We encourage authors to run the migration script themselves to preserve author attribution.

You can migrate all your open pull requests at once, following the steps in the [documentation](https://github.com/spack/migrate-package-prs?tab=readme-ov-file#usage-instructions).

---

*This comment was posted automatically.*

finkeljos commented:
@haampie Not really sure what this is.  For my last PR, I just did the normal procedure. Is it possible to merge the PR without this?
